app/src/main/java/com/spendster/presentation/addExpenses/chooseCategory/CategoryRecyclerViewAdapter.java

```diff
@@ -73,20 +74,18 @@ public void bind(List<Category> categories) {
     }
 
     public static class CategoryViewHolder extends RecyclerView.ViewHolder {
-        TextView categoryTitle;
-        ImageView categoryIcon;
-        CardView cardView;
+        private TextView categoryTitle;
```

Comment:
@SimonMaha for views naming it's nice to use prefix, declaring what is the time, for example, tvTitle, ivIcon etc